x-pack/plugins/timelines/common/ecs/process/index.ts

```diff
@@ -10,6 +10,9 @@ import { Ext } from '../file';
 export interface ProcessEcs {
   Ext?: Ext;
   entity_id?: string[];
+  entry_leader?: ProcessSessionData;
```

Comment:
not sure if it helps, but currently we are using this type (as defined in `x-pack/plugins/session_view/common/types/process_tree/index.ts`)


```js
export interface ProcessSessionData {
  entity_id: string;
  args: string[];
  args_count: number;
  command_line: string;
  executable: string;
  name: string;
  interactive: boolean;
  working_directory: string;
  pid: number;
  start: string;
  end?: string;
  user: User;
  exit_code?: number;
  entry_meta?: EntryMeta;
  tty: Teletype;
}

export interface User {
  id: string;
  name: string;
}

export type EntryMetaType =
  | 'init'
  | 'sshd'
  | 'ssm'
  | 'kubelet'
  | 'teleport'
  | 'terminal'
  | 'console';

export interface Teletype {
  descriptor: number;
  type: string;
  char_device: {
    major: number;
    minor: number;
  };
}


export interface ProcessEcs {
  //...
  parent: ProcessSessionData;
  session_leader: ProcessSessionData;
  entry_leader: ProcessSessionData;
  group_leader: ProcessSessionData;
  //...
}
```